assets/sass/components/global/_googlesitekit-view-only-menu.scss

```diff
@@ -116,5 +114,13 @@
 			// provided by the OptIn component.
 			margin-left: -11px;
 		}
+
+		&--user-can-authenticate {
```

Comment:
@nfmohit Just a note to avoid concatenating the selectors 😁 The thing is that it makes it hard to read and to look for this class name within the code. I know we have it in some places in the code but they are legacy code and should be avoided 👍